.github/workflows/template-lighthouse.yml

```
@@ -65,3 +108,15 @@ jobs:
         run: |
           $report_content='${{ steps.lighthouse-results-generate.outputs.report }}'
           "$report_content" | Out-File -FilePath $env:GITHUB_STEP_SUMMARY -Append
+
+      - name: Commit lighthouse report to repo
+        if: github.event_name != 'pull_request'
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git add prod-lighthouse-report.md
+          git commit -m "Updated Lighthouse markdown report generated by workflow run: ${{ github.run_number }}" || true
```

```suggestion
          git commit -m "Updated Lighthouse markdown report generated by workflow run: ${{ github.run_number }} [skip ci]" || true
```
this may trigger a recursive loop if CI is triggered